examples/cell/src/index.ts

```diff
@@ -1,44 +1,35 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-import { PageConfig, URLExt } from '@jupyterlab/coreutils';
-(window as any).__webpack_public_path__ = URLExt.join(
-  PageConfig.getBaseUrl(),
-  'example/'
-);
```

Comment:
Setting the public path needs to be above the css files, IIRC, so moving it below won't work. Currently this block of code is in index.ts for simplicity in the example. A slightly more complicated, but "better" approach would be to do what we do in jlab itself - put this code in a separate file, and make sure it is included before the main file in the webpack config. For example, see https://github.com/jupyterlab/jupyterlab/blob/master/dev_mode/publicpath.js and https://github.com/jupyterlab/jupyterlab/blob/4cb624755d6ddf4e59b3993fd9fcebaf255ec5cb/dev_mode/webpack.config.js#L298, or see https://github.com/jupyterlab/jupyterlab/blob/master/examples/federated/core_package/publicpath.js and https://github.com/jupyterlab/jupyterlab/blob/4cb624755d6ddf4e59b3993fd9fcebaf255ec5cb/examples/federated/core_package/webpack.config.js#L185

Reply:
(Note that the public path in those two examples is different than the public path set in these examples, so that needs to be thought through, not just blindly copied)

Reply:
Thanks for the review @jasongrout 

I restarted this PR on the latest master not applying the VSCode import sorter on the example (nor on the debugger package) to ensure those specific cases are not modified.